.npmrc

```diff
@@ -0,0 +1,2 @@
+//registry.npmjs.org/:_authToken=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
```

Comment:
Need to add to add this to gitignore as well ... also need to rotate tokens now.